ui/raidboss/data/05-shb/trial/castrum-ex.js

```diff
@@ -0,0 +1,123 @@
+import ZoneId from '../../../../../resources/zone_id.js';
+import NetRegexes from '../../../../../resources/netregexes.js';
+import Conditions from '../../../../../resources/conditions.js';
+import { Responses } from '../../../../../resources/responses.js';
+
+export default {
+  zoneId: ZoneId.CastrumMarinumExtreme,
+  triggers: [
+    {
+      id: 'CastrumEx Optimized Ultima',
+      netRegex: NetRegexes.startsUsing({ source: 'The Emerald Weapon', id: '5B10', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'CastrumEx Aetheroplasm Production',
+      netRegex: NetRegexes.startsUsing({ source: 'The Emerald Weapon', id: '55AA', capture: false }),
+      alertText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Get orbs',
+          cn: '撞球',
+        },
+      },
+    },
+    {
+      id: 'CastrumEx Divide Et Impera P1',
+      netRegex: NetRegexes.startsUsing({ source: 'The Emerald Weapon', id: '555B', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      response: Responses.spread(),
+    },
+    {
+      id: 'CastrumEx Divide Et Impera P2',
+      netRegex: NetRegexes.startsUsing({ source: 'The Emerald Weapon', id: '5537', capture: false }),
+      alertText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          cn: '八方位',
```

Comment:
This can reuse the output strings from `E2S Doomvoid Cleaver`, I think?

Reply:
resolved